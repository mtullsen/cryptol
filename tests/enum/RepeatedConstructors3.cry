enum A = X | Y
enum B = A | X
