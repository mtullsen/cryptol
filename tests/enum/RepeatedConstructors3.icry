:load RepeatedConstructors3.cry
