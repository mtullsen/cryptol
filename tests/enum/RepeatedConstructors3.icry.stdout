Loading module Cryptol
Loading module Cryptol
Loading module Main

[error]
    Overlapping symbols defined:
    (at RepeatedConstructors3.cry:1:10--1:11, Main::X)
    (at RepeatedConstructors3.cry:2:14--2:15, Main::X)
